### ISSUE TITLE
Document browser support

### ISSUE THREAD

**Issue by thijstriemstra** [collab-project/videojs-record#255]
Something like https://www.npmjs.com/package/recordrtc#browsers-support

**Comment by viperfx** [comment]
Yes, please document this. What is the browser support atm? I am assuming firefox and chrome. Does it work with Safari and Edge? 👍

**Comment by thijstriemstra** [comment]
Added info to readme. Also see https://github.com/collab-project/videojs-record/wiki/Browser-support for a script to generate a report about current browser support.
